nuget.config

```diff
@@ -11,15 +11,18 @@
 
   <packageSources>
     <clear />
-    <add key="slvs_input" value="https://sonarsource.pkgs.visualstudio.com/399fb241-ecc7-4802-8697-dcdd01fbb832/_packaging/slvs_input/nuget/v3/index.json" />
+    <!-- SonarSecrets package published by the Language Team -->
+    <add key="repox_releases" value="https://repox.jfrog.io/repox/api/nuget/sonarsource-nuget-releases" />
```

Comment:
Are they signed with the same certificate as the dotnet analyzers? If not, the trustedSigners need to be updated.

Reply:
Sorry, forgot to mention that. The signature is the same. As a test, I commented it out and as expected the NuGet fetch failed.